moksha.common/moksha/common/commands/cli.py

```diff
@@ -45,7 +45,7 @@ class MokshaCLI(object):
 
     def _exec(self, process, *args, **kw):
         args = args and [process] + list(args) or [process]
-        print("Running %s" % ' '.join(args))
+        print("Running %r" % args)
```

Comment:
Since `args` is a tuple, won't it cause issues if it contains more than 1 value?
We might need to make it `% (args,)` to be safe?